config/job_conf.xml

```diff
@@ -54,18 +54,18 @@
             <param id="docker_enabled">true</param>
         </destination>
         <destination id="iso2flux-container" runner="k8s">
-            <param id="docker_repo_override">docker-registry.phenomenal-h2020.eu</param>
+            <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
             <param id="docker_owner_override">phnmnl</param>
             <param id="docker_image_override">iso2flux</param>
-            <param id="docker_tag_override">latest</param>
+            <param id="docker_tag_override">dev_v0.2_cv0.2.23</param>
             <param id="max_pod_retrials">1</param>
             <param id="docker_enabled">true</param>
         </destination>
         <destination id="midcor-container" runner="k8s">
-            <param id="docker_repo_override">docker-registry.phenomenal-h2020.eu</param>
+            <param id="docker_repo_override">container-registry.phenomenal-h2020.eu</param>
             <param id="docker_owner_override">phnmnl</param>
             <param id="docker_image_override">midcor</param>
-            <param id="docker_tag_override">latest</param>
+            <param id="docker_tag_override">dev_v1.0_cv0.2.30</param>
```

Comment:
will tag this and revert to :latest once the CI generates an image. Alternatively I could set up the CI job to build from release as well with another prefix on the tag (like `rc` for release candidate), and then we start with the release 1 process now... what do you think? that would require though fiddling with Pierrick's version tagging scripts. I would rather though start the release process once we have merged your workflow installation part.